docs/Installing.asciidoc

```diff
@@ -18,34 +18,70 @@ assumed that the reader is already familiar with openQA and has already read the
 Starter Guide, available at the
 https://github.com/os-autoinst/openQA[official repository].
 
+
 == Installation
 
-The easiest way to install openQA is from packages. You can find openSUSE
-packages in OBS in the
-https://build.opensuse.org/project/show/devel:openQA[openQA:devel] repository.
+Take in mind that there are a lot of disruptive changes between openQA
+versions, you need to be sure that the webui and the worker that you are using
+have the same version number or, at least, are compatible.
 
-For Fedora, packages are available in the official repositories for Fedora 23
-and later. Installation on these distributions is therefore pretty simple:
+For exampe, the package distributed with Leap 42.3 is not compatible with the
+version on Tumbleweed.
+And the package distributed with Tumbleweed may not be compatible with the
+version in development package.
 
-[source,sh]
-----
-# openSUSE Leap 42.2
-zypper ar -f obs://devel:openQA/openSUSE_Leap_42.2 openQA
-zypper ar -f obs://devel:openQA:Leap:42.2/openSUSE_Leap_42.2 openQA-perl-modules
+=== Installation from distribution packages
 
-# openSUSE Leap 42.3
-zypper ar -f obs://devel:openQA/openSUSE_Leap_42.3 openQA
-zypper ar -f obs://devel:openQA:Leap:42.3/openSUSE_Leap_42.3 openQA-perl-modules
+The easiest way to install openQA is from distribution packages.
 
-# openSUSE Tumbleweed
-zypper ar -f obs://devel:openQA/openSUSE_Tumbleweed openQA
+- For openSUSE, packages are available for Leap 42.3 and Later.
+- For Fedora, packages are available in the official repositories for Fedora 23
+and later.
 
-# all openSUSE
+You can install the packages using these commands.
+
+[source,sh]
+-------------------------------------------------------------------------------
+# openSUSE 42.3+
 zypper in openQA
 
+
 # Fedora 23+
 dnf install openqa openqa-httpd
-----
+-------------------------------------------------------------------------------
+
+
+=== Installation from development packages
+
+You can find the development packages of openQA in OBS in the
+https://build.opensuse.org/project/show/devel:openQA[openQA:devel] repository.
+
+To add the development repository to your system, you can use these commands.
+
+[source,sh]
+-------------------------------------------------------------------------------
+# openSUSE Tumbleweed
+zypper ar -f obs://devel:openQA/openSUSE_Tumbleweed devel-openQA
+
+
+# Leap 42.3
```

Comment:
when in L63 we write "openSUSE Tumbleweed" then this should be consistent -> "openSUSE Leap 42.3"